src/components/map/CountriesTileLayer.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import { useMap } from "react-leaflet";
+import L from 'leaflet';
+import { layerStyle, highlightStyles } from './MapStyle';
+import _ from "lodash";
+import { createAltPopup } from "../../utils/mapUtils"
+import ReactDOMServer from 'react-dom/server';
+// eslint-disable-next-line no-unused-vars
+import VectorGrid from "leaflet.vectorgrid";
+
+const toggleCountrySelection = (e, layer, highlightStyle) => {
```

Comment:
If possible we should create interfaces for these props

Reply:
absolutely not.